kronjob/_kronjob.py

```diff
@@ -110,7 +110,11 @@ def _get_args(*keys):
                 containers=[
                     k8s_models.V1Container(
                         env=env, name=_get_args('containerName')['container_name'],
-                        **_get_args('args', 'command', 'image', 'resources')
+                        resources=k8s_models.V1ResourceRequirements(
+                            limits={'cpu': aggregate_job.get('cpuLimit'), 'memory': aggregate_job.get('memoryLimit')},
```

Comment:
these should be set to sensible defaults - 1 cpu, 1 Gi mem, and same for limits to get guaranteed QoS

Reply:
yes but then it forces authors to always specify resources if they need more than the default (usually true)

maybe adding CLI defaults for resources is an OK compromise - kronjob doesn't provide huge defaults for workloads that don't need it, and we can keep the resource config scoped to CI

Reply:
hmm, seems conceptually easier if there aren't a bunch of cmd line flags that change behaviors. Will add the defaults for now and we can revisit later if it gets annoying